cli/src/commands/init.rs

```diff
@@ -233,13 +234,8 @@ impl Opts {
 
 		// Build the uproject path
 		let current_dir = std::env::current_dir()?;
-		let uproject_path = find_uproject_file(&current_dir)
-			.await
-			.context("find_uproject_file")?
-			.context("could not find *.uproject file")?;
-		let uproject_path_unix = uproject_path
-			.strip_prefix(current_dir)
-			.context("failed to strip uproject path prefix")?
+		let uproject_path = unwrap!(unwrap!(find_uproject_file(&current_dir).await));
+		let uproject_path_unix = unwrap!(uproject_path.strip_prefix(current_dir))
 			.components()
```

Comment:
@NathanFlurry In this change, is any data lost with losing the context string? I think we said context just points to the location for a better backtrace, but is it ever used for other important info?